src/game/Strategic/Campaign_Types.h

```diff
@@ -16,6 +16,15 @@ static inline UINT8 SECTOR(UINT8 const x, UINT8 const y)
 	return (y - 1) * 16 + x - 1;
 }
 
+//Macro to convert sector IDs such as A1,P16 to 0-255
+static inline UINT8 SECTOR_FROM_COORDINATES(const char* coordinates)
```

Comment:
Realized this should be called "Short Sector String". (See StrategicMap.cc:GetShortSectorString))